bundles/org.openhab.binding.hdpowerview/README.md

```diff
@@ -104,6 +104,10 @@ All of these channels appear in the binding, but only those which have a physica
 
 Please note that RSSI values will only be updated upon manual request by a `REFRESH` command (e.g. in a rule).
 
+The channels `secondary` and `vane` only provide useful data if the shade physically supports such channels.
+The binding queries the shade capabilities if they would support such channels, and if necessary it creates those channels automatically.
+The remainin channels in the above table always provide useful data, so they are always created.
```

Comment:
I would either remove this or simplify to just mention that the channels only exist for shade things when supported. However, it's kind of already mentioned in the channel descriptions by "(if any)" so any description here is quite verbose.

Mentioning "usefulness of data" seems too complicated/misleading - either you can read and write values (for example get position and set position) or you can't (if not supported).

Reply:
Ditto